### PR TITLE
#5726 card changes #5727 button space issue

### LINKED ISSUE #5726: Cards small issues
image
- please change the hover effect to correct color, look for example at footer (color change to k-hovergrey)

image
- make the whole card clickable - like the collection card - cursor to pointer in the whole card container

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #5726 #5727
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 4ca5d3c</samp>

This pull request improves the user interface and design of the `NeoNftCard` component by making the whole card clickable and navigable to the nft details page, and by changing the color of the collection name link to match the theme and contrast better on hover. These changes affect the files `NeoNftCard.vue` and `_collection-card.scss`.

### <samp>🤖 Generated by Copilot at 4ca5d3c</samp>

> _`nuxt-link` expands_
> _card becomes more clickable_
> _spring breeze of UX_

### CODE REVIEW

**Comment by exezbcz** [comment]
the nft name should not change with the hover 
canary
image
deploy
image

File: libs/ui/src/components/NeoNftCard/NeoNftCard.vue [hunk 1]
```diff
@@ -9,51 +9,52 @@
...
+            data-cy="card-money" />
+          <span
+            v-if="variant !== 'minimal'"
+            class="chain-name is-capitalized is-size-7"
+            >{{ getChainNameByPrefix(prefix) }}</span
+          >
+        </div>
       </div>
-    </div>
+    </nuxt-link>
```
Comment (roiLeo) [inline libs/ui/src/components/NeoNftCard/NeoNftCard.vue:57]:
```suggestion
    </nuxt-link>
```
We are not using `nuxt` in `libs/ui`

Comment (prachi00) [reply libs/ui/src/components/NeoNftCard/NeoNftCard.vue:57]: it was already there though and seems to work, I replaced it with <a> though now

**Comment by exezbcz** [comment]
> make the whole card clickable - like the collection card - cursor to pointer in the whole card container

**Comment by prachi00** [comment]
> https://deploy-preview-5733--koda-canary.netlify.app

@exezbcz it is clickable though? this part
Screen Shot 2023-04-24 at 9 47 03 AM

**Comment by exezbcz** [comment]
> the nft name should not change with the hover canary image deploy image

yeah, but the text should not change to its hover color 
image

**Comment by prachi00** [comment]
> > the nft name should not change with the hover canary image deploy image
> 
> yeah, but the text should not change to its hover color image

done @exezbcz

**Comment by prachi00** [comment]
can we merge this one now?

**Comment by roiLeo** [comment]
> can we merge this one now?

I still see nuxt-link

**Comment by prachi00** [comment]
> > can we merge this one now?
> 
> I still see nuxt-link

removed from there as well

**Comment by prachi00** [comment]
can we merge this?

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $15 @ 28.13 USD/KSM ~ 0.533 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x8ccefea8b3e56ff306fd35231730e4173e1566d314adc2747419b5163650bb91](https://kusama.subscan.io/extrinsic/0x8ccefea8b3e56ff306fd35231730e4173e1566d314adc2747419b5163650bb91)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**